<launch>

  <group ns="turtlebot1">
    <param name="tf_prefix" value="turtlebot1" />
    <include file="$(find gatlin)/launch/simulator_turtlebot.launch" >
      <arg name="rviz" value="false"/>
      <arg name="gazebo" value="false"/>
      <arg name="init_pose" value="-x 0 -y 0 -z 0.93"/>
      <arg name="robot_name" value="turtlebot1"/>
    </include>
  </group>

  

  <!-- <include file="$(find gatlin)/launch/simulator_baxter.launch" >
    <arg name="gazebo" value="false"/>
    <arg name="init_pose" value="-x 0 -y 0 -z 0.93"/>
  </include> -->

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find gatlin)/rviz/navigation.rviz"/>

  <arg name="world_file"  default="$(env TURTLEBOT_GAZEBO_WORLD_FILE)"/>
  <arg name="gui" default="true" />
  <arg name="sim" default="true"/>

  <param name="/use_sim_time" value="$(arg sim)"/>
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="use_sim_time" value="true"/>
    <arg name="debug" value="false"/>
    <arg name="gui" value="$(arg gui)" />
    <arg name="world_name" value="$(arg world_file)"/>
  </include>

</launch>